### DISCUSSION TITLE
Better README?

### DISCUSSION THREAD

**Discussion by rmorshea** [idom-team/idom#740] [Ideas]
The primary goal of the README is to quickly describe IDOM is as quickly as possible. It should do this in no more that two sentences. If the reader is still interested, the content which follows should then make it easy for the reader to dive into the points of interest that caught their interest just prior.

The [current README](https://github.com/idom-team/idom/blob/e93ae19adc527397eddb8a6c1f3374e2889841cb/README.md) is much better than our old one, but I think there's still room for improvement. In the original [README overhaul](https://github.com/idom-team/idom/pull/705#issue-1163756980), @Archmonger laid down a set of points that we should try and get across. To some degree we touch on all those points now, but I think there's too many things there. The README probably be laser focused on 3-4 concise take-aways.

So, before we try and re-write it though, I'd like to try and figure out what that concise set of points should be.

**Comment by rmorshea** [comment]
I think the following would be my picks, but I'd like some feedback on this:

- Create web apps without Javascript
- Describe the UI with React-like components
- Integrates with web-frameworks like FastAPI and Django

cc: @Archmonger @acivitillo @liberty-rising @sylvoslee

**Reply by Archmonger** [reply]
I personally think the first sentence in the current readme is a bit busy, but I think the rest of it is fairly concise.

I'd still like to hit all our "major features" I outlined in my PR if possible:
- ReactJS based design
- Hooks
- Events
- NPM imports
- JavaScript components
- Multi web framework support
